src/app/teams/teams-view.component.ts

```diff
@@ -184,13 +184,28 @@ export class TeamsViewComponent implements OnInit, OnDestroy {
 
   changeMembership(type, memberDoc?) {
     const changeObject = this.changeObject(type, memberDoc);
-    changeObject.obs.pipe(
+    return {
+      request: changeObject.obs.pipe(
```

Comment:
Please fix the indentation for lines 189-192.

Reply:
thank you now fixed..